sig-scalability/charter.md

```diff
@@ -0,0 +1,192 @@
+them by measuring performance and scalability indicators and publishing the
+results.
+
+We also coordinate and contribute to general system-wide scalability and
+performance improvements (that don’t fall into the charter of another individual
+SIG) as well as provide consultations about any scalability and performance
+related aspects of Kubernetes.
+
+## What can we do/require from other SIGs
+Scalability and performance are horizontal aspects of the system - changes in a
+single place of Kubernetes may affect the whole system. As a result, to
+effectively ensure Kubernetes scales, we need a special cross-SIG privileges.
+
+- We can rollback any merged PR if it has been identified as a cause of an SLO
+  regression. The offending PR should only be merged again after proving to pass
+  tests at scale.
+- We can pause the merge queue in case of a regression observed until a particular
+  PR has been identified as cause of the regression and regression has been
+  mitigated. The “Rules of engagement” of pausing merge-queue and rationale for
+  necessity of its introduce are explained in a separate doc. <br/>
+  TODO(wojtek-t, shyamjvs): Write it down and link here.
+- We require significant changes (in terms of impact, such as: update of etcd,
+  update of Go version, major architectural changes, etc.) may only be merged:
+  - with an explicit approval from a SIG-scalability approver and
+  - after having passed performance testing on biggest supported clusters (unless
+    found unnecessary by scalability approver)
+- We can block a feature from transitioning to Beta status if (when turned on) it
+  causes a significant degradation of overall Kubernetes scalability/performance.
+  (Ideally it would be “SLI degradation of more than X%” or “breaking SLO”, but
+  initially it may also be SIG-scalability decision based on public test results).
+- We can block a feature from transitioning to GA status if it cannot be used at
+  scale.
+- We can require a SIG to introduce a regression-catching benchmark test for a
+  scalability-critical functionality.
+
+For the record, by regression above we mean a regression identified by the set 
+of release-blocking scalability/performance tests (as defined by
+sig-release-master-blocking group of test suites).
+
+## SIG Values
+
+- We are NOT firefighters, we are fire-prevention specialists.
+- We promote deep technical understanding of the Kubernetes system and our tools.
+- We strive to eliminate toil.
+- We work towards building a scalable Kubernetes even in face of superlinear growth
+  of number of contributors.
+
+## Scope and subprojects
+The scope of SIG Scalability covers all aspects of Kubernetes scalability and
+performance. However, all issues that fully fall under a single SIG are implicitly
+delegated to that SIG.
+
+SIG scalability subprojects are as follows.
+
+<table>
+<tr>
+  <td>Subproject</td>
+  <td>Description</td>
+  <td>Example Artifacts</td>
```

Comment:
Can this table have a column for a link to the OWNERS file? It requires a bit of digging for each of these.

Reply:
Done (modulo one line, which I would like to figure out independently in parallel - hopefully that doesn't block charter).